libexec/scoop-bucket.ps1

```diff
@@ -19,19 +19,48 @@
 #     scoop bucket known
 param($cmd, $name, $repo)
 
-. "$psscriptroot\..\lib\core.ps1"
-. "$psscriptroot\..\lib\buckets.ps1"
-. "$psscriptroot\..\lib\help.ps1"
+. "$PSScriptRoot\..\lib\core.ps1"
+. "$PSScriptRoot\..\lib\buckets.ps1"
+. "$PSScriptRoot\..\lib\help.ps1"
 
 reset_aliases
 
 $usage_add = "usage: scoop bucket add <name> [<repo>]"
 $usage_rm = "usage: scoop bucket rm <name>"
 
-switch($cmd) {
+function list_buckets {
+    $buckets = @()
+
+    foreach ($bucket in Get-LocalBucket) {
+        $source = Find-BucketDirectory $bucket -Root
+        $manifests = (
+            Get-ChildItem "$source\bucket" -Force -Recurse -ErrorAction SilentlyContinue |
+            Measure-Object | Select-Object -ExpandProperty Count
+        )
+        $updated = 'N/A'
+        if (Test-Path (Join-Path $source '.git')) {
+            $updated = git_cmd -C "`"$source`"" log --format='%ch' -n 1
+            $source = git_cmd -C "`"$source`"" config remote.origin.url
+        }
```

Comment:
We can use `(Get-Item $source).LastWriteTime` for folders that are not git repos?

Reply:
Yeah, I think that's alright. Better than no time at all.

Reply:
or both with `yyyy-MM-dd HH:mm:ss`?

```
Name     Manifests Source                                     Updated
----     --------- ------                                     -------
1ocal            3 ~\scoop\buckets\1ocal                      2022-02-02 17:48:10
extras        1380 git@github.com:ScoopInstaller/Extras.git   2022-02-03 00:43:11
```

Reply:
That works too 👍